pkg/ccn-lite/Makefile

```diff
@@ -5,7 +5,7 @@ PKG_LICENSE=ISC
 
 .PHONY: all
 
-export RIOT_CFLAGS = $(CFLAGS) $(INCLUDES)
+RIOT_CFLAGS := $(INCLUDES)
```

Comment:
Does this work? Isn't `$(INCLUDES)` empty at this point? Should this be `=` instead of `:=`?

Reply:
As it is called with `make -C pkg/PKGNAME` the variable was already set from environment variable.
However, there is not reasons to set it with `:=` either.